src/components/layout.js

```diff
@@ -32,13 +32,6 @@ const overwrittenTheme = responsiveFontSizes(createMuiTheme({
 	typography: {
 		fontSize: 16,
 		fontFamily: '"Poppins", sans-serif',
-		h1: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h2: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h3: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h4: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h5: { fontFamily: '"Palanquin Dark", sans-serif' },
-		h6: { fontFamily: '"Palanquin Dark", sans-serif' },
-		subtitle1: { fontFamily: '"Palanquin Dark", sans-serif' },
 		body1: { fontFamily: '"Chivo", sans-serif' },
 		button: { fontFamily: '"Chivo", sans-serif' }
```

Comment:
I think we can remove the button override
```suggestion
```

Reply:
This leads to the navbar buttons to be rendered as Chivo:

![image](https://user-images.githubusercontent.com/1538624/86303130-e35bdd80-bbd8-11ea-965e-988649f94c53.png)

I believe they should be Poppins, as suggested by @jamieliu386.

Reply:
I actually like the top as well now that I look at it side by side